docs/resource-schemas.md

```diff
@@ -331,7 +352,40 @@ For the value of the `demoLinks` array, you can set the following keys.
 | `action`  | Determines the action icon.  | Optional | String | `link`  | `download`, `link`                   |
 | `url`     | Link to the resource.        | Required | String | –       | –                                    |
 
-#### Identifiers
+### Identifiers
 
 Libraries and assets have `id`s to uniquely identify each resource and establish relationships
 between resources. More guidance around format and absolute identifiers coming here soon!
+
+### Nav data
+
+Libraries and assets have documentation pages that are specific to their library. The `navData`
```

Comment:
Remove "and assets". I'd do the commit suggest, but then the CI linter would break due to formatting.